boards/arm/stm32f7508_dk/stm32f7508.dtsi

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
.dtsi files belong to dts/.
Also, you should add the following files instead of this one:
- stm32f750.dtsi
- stm32f750X8.dtsi

Reply:
> Thanks for this work. Looks overall good aside the soc part. Please follow the comments and split soc/dtsi introduction part in a dedicated commit.

Thanks for your review.
I followed your comments, I hope it's correct :)

Reply:
@duchesne9al Please take this comment into account.